packages/libsql-core/src/uri.ts

```diff
@@ -90,7 +90,7 @@ const AUTHORITY_RE = (() => {
     const USERINFO = '(?<username>[^:]*)(:(?<password>.*))?';
     const HOST = '((?<host>[^:\\[\\]]*)|(\\[(?<host_br>[^\\[\\]]*)\\]))';
     const PORT = '(?<port>[0-9]*)';
```

Comment:
a suggestion here to make this more readable
```suggestion
    const USERINFO = "((?<username>[^:]*)(:(?<password>.*))?@)?";
    const HOST = "((?<host>[^:\\[\\]]*)|(\\[(?<host_br>[^\\[\\]]*)\\]))";
    const PORT = "(:(?<port>[0-9]*))?";
    return new RegExp("^" + USERINFO + HOST + PORT + "$", "su");
```